AuditRules.cpp

```diff
@@ -1927,6 +1894,19 @@ void merge_rule(AuditRule& drule, const AuditRule& srule) {
     }
 }
 
+// Assumes that drule and srule have the same CanonicalMergeKey() value
+// Returns true if srule is a strict subset (perms or syscalls) of drule
+bool is_subset(const AuditRule& drule, const AuditRule& srule) {
+    AuditRule rule = drule;
```

Comment:
>AuditRule rule = drule; [](start = 4, length = 23)

It looks rule variable is assigned but unused

Reply:
Removed.